### ISSUE TITLE
[Supporting platform - Make code - Share project]: After resizing the page to 200%, Controls present on the 'Share dialog' are truncated.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5468]
### User Experience:
Low vision users who depend on zooming functionality for better readability will be impacted if the content present on the page is not visible on resizing the page. They will not be able to access the share the project controls to share the project.

### Note: 
User credentials should NOT be included in the bug.

### Prerequisite:

- Open system settings -> Display -> Change the 'Scale' value to '100%' and 'Display Resolution' value to mid-range of '1280x720'.
- Now zoom the page to 200%.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of  "Share Project" dialog.
7. Observe whether controls present in the 'Share project' dialog is truncated or not.

### Actual Result:
Issue: After resizing the page to 200%, Controls present on the 'Share dialog' are truncated.

**Observation:** 
Controls like 'X', 'Embed code' etc... which are present on the dialog are truncated and unable to access with both keyboard and mouse.

### Expected Result:
After resizing the page to 200%, Controls present on the 'Share dialog' should be visible clearly without truncation.

### MAS Reference:
[MAS 1.4.4 – Resize Text](https://aka.ms/MAS1.4.4)

ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:

NA

UIA/DOM details available: 

NA

Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.
<img width="960" alt="Normal mode_Share project dialog" src="https://github.com/microsoft/pxt-microbit/assets/158465724/564d34d7-d14c-49f7-b44d-bd9e386a6884">
<img width="644" alt="MAS1 4 10_After resizing the page to 200zoom, Controls present on the 'Share dialog' are truncated" src="https://github.com/microsoft/pxt-microbit/assets/158465724/1b54c1f1-0bcb-4144-bd8f-79e367b0538d">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#AILimited;#MAS1.4.4;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Jan2024;

**Comment by vroopea** [comment]
As checked the issue on URL: https://makecode.microbit.org/. The issue is still repro and severity is fine. #Regressed:04-08-24;InternalReviewed-Apr2024
